main.go

```diff
@@ -77,6 +80,390 @@ func main() {
+						for rows.Next() {
+							var id int
+							var email string
+							var state string
+							if err := rows.Scan(&id, &email, &state); err != nil {
+								log.Fatal(err)
+							}
+							fmt.Printf("%d\t\t%v\t%v\n", id, email, state)
+
+						}
+					} else if cCtx.String("email") != "" {
+						email := cCtx.String("email")
+						fmt.Println("Customer_ID\tEmail\t\t\tState")
+						rows, err := db.QueryContext(ctx, "SELECT * FROM Customers WHERE Email LIKE CONCAT('%', ?, '%')", email)
+						if err != nil {
+							log.Fatal(err)
+						}
+						defer rows.Close()
+
+						for rows.Next() {
+							var id int
+							var email string
+							var state string
+							if err := rows.Scan(&id, &email, &state); err != nil {
+								log.Fatal(err)
+							}
+							fmt.Printf("%d\t\t%v\t%v\n", id, email, state)
+
+						}
+					} else if cCtx.String("state") != "" {
+						state := cCtx.String("state")
+						fmt.Println("Customer_ID\tEmail\t\t\tState")
+						rows, err := db.QueryContext(ctx, "SELECT * FROM Customers WHERE State LIKE CONCAT('%', ?, '%')", state)
+						if err != nil {
+							log.Fatal(err)
+						}
+						defer rows.Close()
+
+						for rows.Next() {
+							var id int
+							var email string
+							var state string
+							if err := rows.Scan(&id, &email, &state); err != nil {
+								log.Fatal(err)
+							}
+							fmt.Printf("%d\t\t%v\t%v\n", id, email, state)
+
+						}
+					} else {
+
+						fmt.Println("Customer_ID\tEmail\t\t\tState")
+						rows, err := db.QueryContext(ctx, "SELECT * FROM Customers")
+						if err != nil {
+							log.Fatal(err)
+						}
+						defer rows.Close()
+
+						for rows.Next() {
+							var id int
+							var email string
+							var state string
+
+							if err := rows.Scan(&id, &email, &state); err != nil {
+								log.Fatal(err)
+							}
+							fmt.Printf("%d\t\t%v\t%v\n", id, email, state)
+
+						}
+					}
+					return nil
+				},
+			},
+			{
+				Name:  "show-orders",
+				Usage: "displays all the orders within the orders database with an optional customerId and productId filter",
+				Flags: []cli.Flag{
+					&cli.StringFlag{
+						Name:  "customer-id",
+						Usage: "the customer-id of the customer",
+					},
+					&cli.StringFlag{
+						Name:  "product-id",
+						Usage: "the product-id of the product",
+					},
+				}, Action: func(cCtx *cli.Context) error {
+					if cCtx.String("customer-id") != "" && cCtx.String("product-id") != "" {
+						cId := cCtx.String("customer-id")
+						pId := cCtx.String("product-id")
+
+						fmt.Println("Order_ID\tCustomer_ID\t\t\tProduct_ID")
+						rows, err := db.QueryContext(ctx, "SELECT * FROM ORDERS WHERE customer_id LIKE CONCAT('%', ?, '%') AND product_id LIKE CONCAT('%', ?, '%')", cId, pId)
+						if err != nil {
+							log.Fatal(err)
+						}
+						defer rows.Close()
+
+						for rows.Next() {
+							var id int
+							var date sql.NullString
+							var cId string
+							var pId string
+							var sku sql.NullString
```

Comment:
[nitpick] in go it's best-practice to capitalize abbreviation portions of variable: `cID`, `pID` (`sku` is correct since variables should start with lowercase and the entire abbreviation should stay in the same case). Other examples are:
```
userDB
exampleURL
insertSTMT 
```